internal/distro/fedora/distro.go

```diff
@@ -16,63 +16,74 @@ import (
 	"github.com/osbuild/osbuild-composer/internal/rpmmd"
 )
 
-const nameTmpl = "fedora-%s"
-const modulePlatformIDTmpl = "platform:f%s"
+const (
+	// package set names
 
-// The second format value is intentionally escaped, because the
-// format string is substituted in two steps:
-//  1. on distribution level when being created
-//  2. on image level
-const ostreeRefTmpl = "fedora/%s/%%s/iot"
+	// build package set name
+	buildPkgsKey = "build-packages"
 
-const f33ReleaseVersion = "33"
-const f34ReleaseVersion = "34"
-const f35ReleaseVersion = "35"
-const f36ReleaseVersion = "36"
+	// main/common os image package set name
+	osPkgsKey = "packages"
+
+	// blueprint package set name
+	blueprintPkgsKey = "blueprint"
+
+	nameTmpl             = "fedora-%s"
+	modulePlatformIDTmpl = "platform:f%s"
+
+	// The second format value is intentionally escaped, because the
+	// format string is substituted in two steps:
+	//  1. on distribution level when being created
+	//  2. on image level
+	ostreeRefTmpl = "fedora/%s/%%s/iot"
+
+	// Supported Fedora versions
+	f33ReleaseVersion = "33"
+	f34ReleaseVersion = "34"
+	f35ReleaseVersion = "35"
+	f36ReleaseVersion = "36"
+)
 
 type distribution struct {
 	name             string
 	releaseVersion   string
 	modulePlatformID string
 	ostreeRefTmpl    string
 	arches           map[string]architecture
-	buildPackages    []string
 }
 
 type architecture struct {
 	distro             *distribution
 	name               string
 	bootloaderPackages []string
```

Comment:
This should be also removed in favor of using the `packageSetFunc()` and be moved to the `package_sets.go` as I commented in the `PackageSets()` method.